### PR TITLE
Adding TestResultCoordinator file structure

### PR DESCRIPTION
**Pull request by dylanbronson**
Adding TestResultCoordinator file structure - bare bones.

This is under test/modules path. 

It is being created right now for connectivity tests, but the idea is that it will be generic for all tests. So it will go in a more generic folder path.

The TestResultCoordinator will parse test result data based on what type of test result it receives, generate test reports, and send them to the cloud.

### CODE REVIEW

File: Microsoft.Azure.Devices.Edge.sln [hunk 1]
```diff
@@ -197,11 +197,15 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Devices.Edg
...
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ModuleRestarter", "edge-modules\ModuleRestarter\ModuleRestarter.csproj", "{2C7AE89A-648C-4EBC-8084-0E59BAFF496D}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ModuleRestarter", "edge-modules\ModuleRestarter\ModuleRestarter.csproj", "{2C7AE89A-648C-4EBC-8084-0E59BAFF496D}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Relayer", "edge-modules\Relayer\Relayer.csproj", "{C1FDA6CF-7EF6-4C3F-81E7-4C13EE399BF1}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Relayer", "edge-modules\Relayer\Relayer.csproj", "{C1FDA6CF-7EF6-4C3F-81E7-4C13EE399BF1}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "modules", "modules", "{F921339B-32F9-4BF3-B364-2DB01FA2F1A1}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestResultCoordinator", "test\modules\TestResultCoordinator\TestResultCoordinator.csproj", "{8181EB49-62CE-495B-8078-08DCF8C30541}"
```
Comment (dylanbronson) [inline Microsoft.Azure.Devices.Edge.sln]: Everything in this file is auto-generated. I didn't touch any of the other modules or the file structure

File: test/modules/TestResultCoordinator/TestResultCoordinator.csproj [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```
Comment (philipktlin) [inline test/modules/TestResultCoordinator/TestResultCoordinator.csproj]: I think at least you have the changes:
1. handle to build .net core 2.1 and 3.0
2. stylecop
3. reference to ModuleLib

File: test/modules/TestResultCoordinator/Program.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // TODO: Add Coordinator logic
```
Comment (philipktlin) [inline test/modules/TestResultCoordinator/Program.cs]: I think you at least should add shutdownhandle.init call

File: test/modules/TestResultCoordinator/TestResultCoordinator.csproj (hunk 2 above)
Comment (dylanbronson) [reply test/modules/TestResultCoordinator/TestResultCoordinator.csproj]: Done

File: test/modules/TestResultCoordinator/Program.cs (hunk 3 above)
Comment (dylanbronson) [reply test/modules/TestResultCoordinator/Program.cs]: Done

File: test/modules/TestResultCoordinator/Program.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+    class Program
+    {
+        static readonly ILogger Logger = ModuleUtil.CreateLogger("TestResultCoordinator");
+
+        static void Main(string[] args)
+        {
+            // TODO: Add Coordinator logic
+            (CancellationTokenSource cts, ManualResetEventSlim completed, Option<object> handler) = ShutdownHandler.Init(TimeSpan.FromSeconds(5), Logger);
+        }
```
Comment (philipktlin) [inline test/modules/TestResultCoordinator/Program.cs]:
```suggestion
            await cts.Token.WhenCanceled();

            completed.Set();
            handler.ForEach(h => GC.KeepAlive(h));
            Console.WriteLine("SimulatedTemperatureSensor Main() finished.");
            return 0;
        }
```

File: test/modules/TestResultCoordinator/TestResultCoordinator.csproj [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    <LangVersion>latest</LangVersion>
+    <Configurations>Debug;Release;CheckInBuild</Configurations>
+    <HighEntropyVA>true</HighEntropyVA>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <Content Include="docker*/**/*.*" CopyToPublishDirectory="Always" />
+  </ItemGroup>
+
+  <ItemGroup>
```
Comment (philipktlin) [inline test/modules/TestResultCoordinator/TestResultCoordinator.csproj]: remove all package and project reference.

Comment (philipktlin) [reply test/modules/TestResultCoordinator/TestResultCoordinator.csproj]: remove unused references only.

**Pull request merged**